install/kubernetes/helm/meshery/Chart.yaml

```diff
@@ -75,14 +75,9 @@ dependencies:
     version: 0.5.0
     condition: meshery-app-mesh.enabled
   - name: meshery-operator
-    version: 0.5.0
+    version: 0.6.0
     condition: meshery-operator.enabled
-  - name: meshery-broker
-    version: 0.5.0
-    condition: meshery-broker.enabled
-  - name: meshery-meshsync
-    version: 0.5.0
-    condition: meshery-meshsync.enabled
+    repository: "file://../meshery-operator"
```

Comment:
Is there any other way we can reference the repo with Artifact hub link?

Reply:
The official repo for published Meshery Charts is https://meshery.io/charts. Artifact Hub points people to this same repo.